trunk/examples/programs/FloatingPoint/regression/c/.skip

```diff
@@ -0,0 +1,18 @@
+// Timeout
```

Comment:
Can we handle them by increasing the timeout?

Reply:
Increasing the timeout did not help